pkg/apim/APIMUtility.go

```diff
@@ -51,3 +66,19 @@ func (apim *Bundle) IsPayloadJSON() bool {
 	var js json.RawMessage
 	return json.Unmarshal([]byte(apim.Payload), &js) == nil
 }
+
+func (odataFilters *OdataParameters) MakeOdataQuery() (string, error) {
+
+	odataFiltersIt := reflect.ValueOf(odataFilters).Elem()
+	typeOfS := odataFiltersIt.Type()
+	urlParam := url.Values{}
+	for i := 0; i < odataFiltersIt.NumField(); i++ {
+		structVal := fmt.Sprintf("%v", odataFiltersIt.Field(i).Interface())
+		if structVal != "" {
+			urlParam.Set(strings.ToLower(typeOfS.Field(i).Name), structVal)
+		}
+	}
+	resultQuery := "?" + urlParam.Encode()
+	resultQuery = strings.ReplaceAll(resultQuery, "&", "&$")
```

Comment:
This can be done in one line:
```suggestion
	resultQuery := "?" + strings.ReplaceAll(urlParam.Encode(), "&", "&$")
```

Reply:
done